aws/aws.go

```diff
@@ -19,6 +19,15 @@ type Credentials struct {
 	Expiration      time.Time
 }
 
+type ProfileStatus struct {
```

Comment:
If `Profiles` is a slice of `ProfileStatus`-es, I think we should rename `type ProfileStatus` to `Profile`.